contributors/design-proposals/scheduling/taint-node-by-condition.md

```diff
@@ -18,20 +18,20 @@ In addition to this, with taint-based-eviction, the Node Controller already tain
 | ConditionType      | Condition Status   |Effect        | Key      |
 | ------------------ | ------------------ | ------------ | -------- |
 |Ready               |True                | -            | |
-|                    |False               | NoExecute    | node.kubernetes.io/notReady           |
-|                    |Unknown             | NoExecute    | node.kubernetes.io/unreachable        |
-|OutOfDisk           |True                | NoSchedule   | node.kubernetes.io/outOfDisk          |
+|                    |False               | NoExecute    | node.kubernetes.io/not-ready           |
+|                    |Unknown             | NoExecute    | node.alpha.kubernetes.io/unreachable        |
```

Comment:
why not also remove alpha for `unreachable`? so we will handle `not-ready` and `unreachable` in the same way :).

Reply:
Yes, we should, bad rebase.

Reply:
Fired up https://github.com/kubernetes/kubernetes/issues/54198 to track this inconsistency

Reply:
great！